.github/workflows/ci-cd.yaml

```diff
@@ -209,6 +209,8 @@ jobs:
           echo "VITE_API_URL=${{ secrets.VITE_API_URL }}" >> frontend/.env.staging
           echo "VITE_ENVIRONMENT=${{ secrets.VITE_ENVIRONMENT }}" >> frontend/.env.staging
           echo "VITE_SENTRY_DSN=${{ secrets.SENTRY_DSN }}" >> frontend/.env.staging
+          echo "VITE_SENTRY_RELEASE=$(echo ${{ github.sha }} | cut -c1-7)" >> frontend/.env.staging
+          echo "VITE_SENTRY_ENVIRONMENT=staging" >> frontend/.env.staging
```

Comment:
We can use the existing environment variable for that, no need to introduce a new one for Sentry.